templates/index.html

```diff
@@ -21,6 +21,24 @@ <h1>_coming&ZeroWidthSpace;_soon</h1>
   automatically generated from the latest development version each night, so
   they may be potentially unstable.
 
+  <div id="download">
+    <h2>Download</h2>
+
+    <div><img src="icons8-linux-52.png" class="icon" /> <a href="https://download.de-game.org/x86_64-unknown-linux-gnu/nightly.zip">Linux<br/>(<code>x86_64-unknown-linux-gnu</code>)</a></div>
+    <div><img src="icons8-windows-10-50.png" class="icon" /><a href="https://download.de-game.org/x86_64-pc-windows-gnu/nightly.zip">Windows<br/>(<code>x86_64-pc-windows-gnu</code>)</a></div>
+    <div><img src="icons8-mac-50.png" class="icon" /><a href="https://download.de-game.org/aarch64-apple-darwin/nightly.zip">macOS with M series<br/>(<code>aarch64-apple-darwin</code>)</a></div>
+    <div><img src="icons8-mac-50.png" class="icon" /><a href="https://download.de-game.org/x86_64-apple-darwin/nightly.zip">macOS with Intel<br/>(<code>x86_64-apple-darwin</code>)</a></div>
+
+    <div id="download-steps">
+      <h3>Download steps</h3>
+      <ol>
+        <li>Download the ZIP file for your OS and CPU:</li>
+        <li>Extract the ZIP file.</li>
+        <li>Execute binary file called <code>de</code> or <code>de.exe</code>.</li>
+      </ol>
+    </div>
+  </div>
+
   <ol>
```

Comment:
Please remove this part as it is superseded with the above.

Reply:
done